libs/core/langchain_core/runnables/base.py

```diff
@@ -438,12 +438,63 @@ def pipe(
         *others: Union[Runnable[Any, Other], Callable[[Any], Other]],
         name: Optional[str] = None,
     ) -> RunnableSerializable[Input, Other]:
-        """Compose this runnable with another object to create a RunnableSequence."""
+        """Compose this runnable with another object to create a RunnableSequence.
+        .. code-block:: python
+
+                from langchain_community.chat_models.fake import FakeListChatModel
+                from langchain_community.llms.fake import FakeStreamingListLLM
+                from langchain_core.prompts import SystemMessagePromptTemplate
+                from langchain_core.runnables import RunnablePassthrough
+
+                prompt = (
+                        SystemMessagePromptTemplate
+                        .from_template("You are a nice assistant.")+ "{question}"
+                )
+                chat_res = "i'm a chatbot"
+                llm_res = "i'm a textbot"
+                chat = FakeListChatModel(responses=[chat_res], sleep=0.01)
+                llm = FakeStreamingListLLM(responses=[llm_res], sleep=0.01)
+                chain_pick_one = (prompt | {
+                    "chat": chat.bind(stop=["Thought:"]),
+                    "llm": llm,
+                    "passthrough": RunnablePassthrough(),
+                }).pipe(llm)
+
+                print(chain_pick_one.output_schema.schema())
+
+                # {'title': 'FakeStreamingListLLMOutput', 'type': 'string'}
+
+        """
         return RunnableSequence(self, *others, name=name)
 
     def pick(self, keys: Union[str, List[str]]) -> RunnableSerializable[Any, Any]:
         """Pick keys from the dict output of this runnable.
-        Returns a new runnable."""
+        Returns a new runnable.
+        .. code-block:: python
+
+            from langchain_community.chat_models.fake import FakeListChatModel
```

Comment:
Similar comment here, I would try to keep the examples either useful for a real task or minimal, e.g.,
```python
from langchain_community.llms.fake import FakeStreamingListLLM
from langchain_core.output_parsers import JsonOutputParser, StrOutputParser

model = FakeStreamingListLLM(responses=['["a", "b", "c"]'])

chain = (
    model
    | {
        "text": StrOutputParser(),
        "json": JsonOutputParser(),
    }
)
type(chain.pick("json").invoke("bla"))

# list
```
consider stripping away some of the elements of this that aren't necessary (e.g., `stop` kwargs, `sleep`, prompt template, etc)